test/k8s-integration/main.go

```diff
@@ -368,7 +368,10 @@ func handle() error {
 
 	// Run the tests using the testDir kubernetes
 	if len(*storageClassFile) != 0 {
-		err = runCSITests(*platform, pkgDir, testDir, *testFocus, testSkip, *storageClassFile, *snapshotClassFile, cloudProviderArgs, *deploymentStrat)
+		storageClasses := strings.Split(*storageClassFile, ",")
+		for _, scFile := range storageClasses {
+			err = runCSITests(*platform, pkgDir, testDir, *testFocus, testSkip, scFile, *snapshotClassFile, cloudProviderArgs, *deploymentStrat)
```

Comment:
How do the test artifacts look? Do all the test results get aggregated, overwritten, or do they look like 2 different runs?

Reply:
It looks like two different runs in terms of stdout. I'm not sure about the test artifacts, I'll have to look to see how they appear locally.

Reply:
The control plane logs are good, it's everything since the cluster was brought up AFAICT.

Not sure about the junit*.xml logs. Continuing to dig.

Reply:
We can setup the ARTIFACTS env variable and run one of the jobs in run-k8s-integration-local.sh and check if the logs are overwritten for the subsequent run of kubetest.
https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/test/k8s-integration/main.go#L461

Reply:
pull-gcp-compute-persistent-disk-csi-driver-kubernetes-integration should be running these changes

Reply:
Yeah, that's what I'm doing. It looks like the junit stuff is overwritten but I have a fix being tested (but does prow/testgrid care where the junit*.xml files end up?)